### ISSUE TITLE
Inconsitency in use of pick lists (REGManager)

### ISSUE THREAD

**Issue by FreeUKGenIssues** [FreeUKGen/FreeUKRegProductIssues#612]
Issue reported by **REGManager** at 2015-11-12 13:45:01 UTC
Time: 2015-11-12T13:41:02+00:00
Session ID: d95bc52164af44f06264a08e28bbd407
Problem Page URL: [/manage_resources/new](/manage_resources/new)
Previous Page URL: [http://www.freereg.org.uk/feedbacks/new?utf8=%E2%9C%93&feedback_time=2015-11-12+13%3A37%3A34+UTC&session_id=d95bc52164af44f06264a08e28bbd407&user_id=REGManager&problem_page_url=%2Fmanage_resources%2Fnew&previous_page_url=http%3A%2F%2Fwww.freereg.org.uk%2Frefinery%2Flogin&feedback_type=issue](http://www.freereg.org.uk/feedbacks/new?utf8=%E2%9C%93&feedback_time=2015-11-12+13%3A37%3A34+UTC&session_id=d95bc52164af44f06264a08e28bbd407&user_id=REGManager&problem_page_url=%2Fmanage_resources%2Fnew&previous_page_url=http%3A%2F%2Fwww.freereg.org.uk%2Frefinery%2Flogin&feedback_type=issue)
Reported Issue:
Some pick lists require you to also hit the "Select" button and some not.  I assume this is because some let you choose more than one entry, the one that goes immediately does not.  May I suggest that if the option to select more than one is there, it says so, and if not then there should be no need to also "Select".  E    

![Screenshot](http://www.freereg.org.uk)

**Comment by AlOneill** [comment]
Do you mean _drop-down_ lists, such as when picking a County on the Transcriptions (formerly Database Contents) page or after selecting the Manage counties action? Or do you mean other kinds of list?

Please give a specific example of each type of inconsistent 'pick list' -- one that has a Select button and one that does not. Thanks.

**Comment by edickens** [comment]
```
Hi Alison.
Kirk has told me that he uses "Select" for researchers drop-down
lists, but acts when clicking on selection for those who have logged
in.  It is not really important.  I think any inconsistencies should
be tidied up in V2.0  E

On 21/11/2015 00:04, AlOneill wrote:


  Do you mean drop-down lists, such as when picking a
    County on the Transcriptions (formerly Database Contents) page
    or after selecting the Manage counties action? Or do you mean
    other kinds of list?
  Please give a specific example of each type of inconsistent
    'pick list' -- one that has a Select button and one that does
    not. Thanks.
  —
    Reply to this email directly or view
      it on GitHub.
```

**Comment by Captainkirkdawson** [comment]
Closed in favour of #1011
